### PR TITLE
Fabric 7.0: update the redDark color in the default palette.

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Description of changes

Updating the `redDark` default palette color slot with the new fluent value agreed with designers (@betrue-final-final to confirm). This color is used for errors.
Update the autogenerated scss files too to match the new colors.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8563)

### CODE REVIEW

**Comment by Vitalius1** [comment]
@phkuo, @JasonGore can I please get your review for this change so I can merge it and update my next PR? Thanks!
@jdhuntington where is the screener test?

File: packages/office-ui-fabric-react/src/common/_semanticSlots.scss [hunk 1]
```diff
@@ -40,14 +40,14 @@ $buttonBackgroundHoveredColor: '[theme:buttonBackgroundHovered, default: #edebe9
 $buttonBackgroundCheckedHoveredColor: '[theme:buttonBackgroundCheckedHovered, default: #edebe9]'; 
 $buttonBackgroundPressedColor: '[theme:buttonBackgroundPressed, default: #edebe9]'; 
 $buttonBackgroundDisabledColor: '[theme:buttonBackgroundDisabled, default: #f3f2f1]'; 
-$buttonBorderColor: '[theme:buttonBorder, default: transparent]';
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: wait why did this change?

Comment (phkuo) [reply packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: oh nm this is the 7.0 branch

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: Yeah. This was something that was not picked up with the first change of the neutral colors due to you changing at the same time the script that auto-generates these files. Now it's actually picking it up.

**Pull request merged**